assets/src/edit-story/components/canvas/karma/rightClickMenu.karma.js

```diff
@@ -284,4 +325,232 @@ describe('Right Click Menu integration', () => {
       expect(pages[0].elements[0].isBackground).toBe(true);
     });
   });
+
+  describe('right click menu: foreground media', () => {
+    it('should set media as the background', async () => {
+      const earthImage = await addEarthImage();
+
+      // right click image
+      await rightClickOnTarget(
+        fixture.editor.canvas.framesLayer.frame(earthImage.id).node
+      );
+
+      // set image as page background
+      await fixture.events.click(
+        fixture.editor.canvas.rightClickMenu.setAsPageBackground
+      );
+
+      // verify the image has been set as the background
+      // verify the correct page was deleted
```

Comment:
nit / just me: this is a little confusing to me, i get that the existing background is deleted but calling it a page in my mind caused a bit of confusion.

Reply:
That should not be there - nice catch!